src/select-menu/src/OptionsList.js

```diff
@@ -296,7 +295,7 @@ OptionsList.propTypes = {
   onFilterChange: PropTypes.func,
   hasFilter: PropTypes.bool,
   optionSize: PropTypes.number,
-  renderItem: PropTypes.func,
+  itemRenderer: PropTypes.func,
```

Comment:
Worth noting that this is a breaking change (although it doesn't matter given that v6 is a lot of breaking changes). To minimize API churn, we could potentially keep this named `renderItem` -- what do you think? 

Otherwise, would be good to add this entry into the `v6-migration-guide`

Reply:
I don't think this is really a breaking change, as this prop isn't actually exposed anywhere. Could rename to renderItem if you disagree though

Reply:
We technically do export `OptionsList` - it's just not documented on the docs site 😬 . Let's keep it as `renderItem` to minimize the code churn and paper-cut upgrade path, if that's okay?